src/test/java/core/basesyntax/sevrice/impl/CsvFruitTransactionParserImplTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.sevrice.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.sevrice.CsvFruitTransactionParser;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CsvFruitTransactionParserImplTest {
+    private static final int FIRST_ELEMENT_INDEX = 0;
+    private static final Fruit APPLE = new Fruit("apple");
+    private static final FruitTransaction APPLE_BALANCE_OPERATION
+            = new FruitTransaction(FruitTransaction.Operation.BALANCE,
+            APPLE, 100);
+    private static final String FIRST_STRING = "this,must,deleted";
+    private final List<FruitTransaction> transactions = new ArrayList<>();
+    private final List<String> testList = new ArrayList<>();
+    private final CsvFruitTransactionParser csvFruitTransactionParser
+            = new CsvFruitTransactionParserImpl();
```

Comment:
![image](https://user-images.githubusercontent.com/81781479/195616887-db655b53-3b68-4ab3-82c4-59731e66bf77.png)

Reply:
not fixed